docs/getting_started/install-on-mac-os-and-windows.md

```diff
@@ -0,0 +1,98 @@
+# Installation on macOS or Windows
```

Comment:
Actually, "Install Ibexa DXP on macOS or Windows" would be best, just like in the regular installation page.

The page is being moved *out* of the contributing section (that was a really wrong location for it anyway), but you're right about unifying page names in another PR.